docs/plone_training_config/instructions.md

```diff
@@ -247,11 +245,11 @@ If you are completely new to node and companions, please see [Volto Documentatio
    node -v
```

Comment:
Line 244 is not properly indented and needs one more leading space to align properly.